### PR TITLE
TASK: Add current state

### PR DESCRIPTION
**Pull request by DanielSiepmann**

### CODE REVIEW

File: Classes/Log/Writer/Console.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ */
+
+use TYPO3\CMS\Core\Log\Writer\AbstractWriter;
+use TYPO3\CMS\Core\Log\LogLevel;
```
Comment (justusmoroni) [inline Classes/Log/Writer/Console.php]: This is not used in this class.

File: Classes/Tests/UnitTestCase.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ */
+
+
```
Comment (justusmoroni) [inline Classes/Tests/UnitTestCase.php]: Remove duplicate line.

File: Documentation/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+build/
```
Comment (justusmoroni) [inline Documentation/.gitignore]: Shouldn't this be moved to the global .gitignore file?

File: Documentation/source/conf.py [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = ['_static']
+
+
+# -- Options for HTMLHelp output ------------------------------------------
+
+# Output file base name for HTML help builder.
+htmlhelp_basename = 'CodappixTYPO3CoreExtensiondoc'
```
Comment (justusmoroni) [inline Documentation/source/conf.py]: `CodappixTYPO3CoreExtensiondoc`

Is this correctly named here?

Check everywhere.

File: Documentation/source/index.rst [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+The extension provides two further log writer for TYPO3:
+
+Console
+    Will output all log entries without buffering to `stdout`.
+    Output will not be styled in any way. Messages are displayed plain, prefixed with log level as
+    string representation.
+
+AnsiConsole
+    Will output all log entries without buffering to `stdout`.
+    Output will not styled. Messages are displayed colored depending on their log level.
```
Comment (justusmoroni) [inline Documentation/source/index.rst]: `Output will not styled. ` => `Output will not be styled.`

**Review by justusmoroni — CHANGES_REQUESTED** [review]

File: Documentation/.gitignore (hunk 3 above)
Comment (DanielSiepmann) [reply Documentation/.gitignore]: No, as we use this folder for development in `composer.json`.
Everyone who will work on this extension and runs `composer install` will have this folder which should be ignored. So it's a project folder.

Comment (justusmoroni) [reply Documentation/.gitignore]: Sorry i meant the .gitignore file in the root dir of the extension.

This .gitignore file is in the Documentation folder.

Comment (DanielSiepmann) [reply Documentation/.gitignore]: Fixed

**Review by justusmoroni — APPROVED** [review]

**Pull request merged**